tests/libnet/hotplug.go

```diff
@@ -132,10 +137,7 @@ func interfaceStatusFromInterfaces(queueCount int32, ifaces []v1.Interface) []v1
 				vmispec.InfoSourceDomain, vmispec.InfoSourceGuestAgent, vmispec.InfoSourceMultusStatus),
 			QueueCount:       queueCount,
 			PodInterfaceName: namescheme.GenerateHashedInterfaceName(iface.Name),
-		}
-
-		if iface.SRIOV == nil {
```

Comment:
Please note that this code is also called from the SR-IOV hotplug test.
For SR-IOV the link state is always empty.